ui/components/ErrorBoundary.js

```diff
@@ -1,41 +1,28 @@
-import { Button } from '@material-ui/core';
+import { Button } from '@mui/material';
 import React from 'react';
+import { ErrorBoundary as SistentErrorBoundary } from '@sistent/react';
 
 /**
  * ErrorBoundary is a React component that catches JavaScript errors in its child components and renders a fallback UI when an error occurs.
  * It should be used as a wrapper around components that might throw errors.
  * @deprecated use error boundary from sistent instead
  */
-class ErrorBoundary extends React.Component {
```

Comment:
seems like we can get ride of this comment in the comments now: `@deprecated use error boundary from sistent instead`

Reply:
done